system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -1589,6 +1594,13 @@ LDAPSECURITY_API IAuthMap *newDefaultAuthMap(IPropertyTree* config)
             {
                 StringBuffer pathstr, rstr;
                 location->getProp("@path", pathstr);
+                location->getProp("@resource", rstr);
+
+                if(rstr.length() == 0)
```

Comment:
Slightly concerned that resource name is not used except for tracing output - is that correct?

Reply:
There are no instances of a defaultAuthMap in the platform code, but it appears that "resource" is not needed.   I refactored this code to only query resource name if the pathstr is missing.